### PR TITLE
Pieplot

### LINKED ISSUE #138: Backend: Need pieplot class in plot.data
Very similar to barplot class, but no barmode arg, fewer var inputs it recognizes and it should aggregate values beyond the top 7 if there are more than 8 unique values.

### PR DESCRIPTION
**Pull request by d-callan**
resolves #138

### CODE REVIEW

**Comment by d-callan** [comment]
conceptually i think its all there, and passing unit tests. want to set up a local backend yet using this and veupathdb/edadataservice#150 together before merging though..

File: R/class-plotdata-pie.R [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+#' @return character name of json file containing plot-ready data
+#' @export
+pie <- function(data, 
+                map, 
+                value = c('count', 'proportion'), 
+                evilMode = c(FALSE, TRUE),
+                verbose = c(TRUE, FALSE)) {
+
+  verbose <- veupathUtils::matchArg(verbose)
+
```
Comment (asizemore) [inline R/class-plotdata-pie.R:197]: matchArg for evilMode and value as well?

File: R/class-plotdata.R [hunk 2]
```diff
@@ -76,7 +76,8 @@ newPlotdata <- function(.dt = data.table(),
     .dt <- data.table::setDT(panelData[[1]])
     panel <- panelData[[2]]
     if (!is.null(panel)){
-      if (uniqueN(.dt[[panel]]) > 25) stop("Maximum number of panels allowed is 25.")
+      # pie, our outlier, is breaking this rule in the map
```
Comment (asizemore) [inline R/class-plotdata.R:79]: Oh pie, you're so great :)

**Review by asizemore — APPROVED** [review]
Sweet! Super pumped for the pie plot!

**Pull request merged**